scripts/compute_bias_metric.py

```diff
@@ -3,7 +3,18 @@
 import numpy as np
 from sklearn.metrics import roc_auc_score
 import argparse
-from utils import compute_auc, compute_subgroup_auc
+
+
+def compute_auc(y_true, y_pred):
```

Comment:
This is now defined twice (also in compute_language_breakdown.py) so should probably keep it in one place

Reply:
well yes, but then I would consider moving it to the detoxify package, as having it here in script won't work as it is not a package available for import (missing `__init__.py` file and I would keep it such)